hnn_core/cell_response.py

```diff
@@ -476,3 +540,59 @@ def read_spikes(fname, gid_ranges=None):
         cell_response.update_types(gid_ranges)
 
     return cell_response
+
+
+def read_spikes_hdf5(fname):
```

Comment:
private function ... once you expose something as public function, you have to maintain that forever.